packages/cli/lib/services/compile.service.ts

```diff
@@ -200,6 +200,10 @@ async function compile({
     const syncConfig = [...providerConfiguration.syncs, ...providerConfiguration.actions].find((sync) => sync.name === file.baseName);
     const type = syncConfig?.type || 'sync';
 
+    if (process.env['_NANGO_IN_REPO']) {
+        file.inputPath = file.inputPath.split('/').slice(0, -1).join('/') + `/${type}s/` + file.baseName + '.ts';
+    }
+
```

Comment:
not a huge fan to ship code to customer that would only be used for dev work but I understand the pain of having to move the built-in integrations during dev work. Can we have the bash script below create a temporary folder structure expected by nango and do the cleanup). Like this we don't have to have any code changes and the hack is self-contained in the bash script?

Reply:
Yeah, I was trying to avoid that initially but I agree in a sense it feels dirty to mess with the logic just for our internal usage.

Updated to create a temp directory like you suggested.